pkg/kopia/repositoryserver/README.md

```diff
@@ -0,0 +1,21 @@
+### Steps to auto-generate code using code-generator and controller-gen tools
```

Comment:
I think all these files should be on the same level as the main controller and not under `kopia`. Also, I think you can remove `bin/controller` binary file.